docker-compose/wildfly-manager/docker-compose.yml

```diff
@@ -0,0 +1,246 @@
+        source: ui_logs
+        target: /usr/local/tomcat/logs
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+    labels:
+#      - traefik.http.routers.apimanui-router.tls=true
+      - traefik.http.routers.apimanui-router.entrypoints=http
+      - traefik.http.routers.apimanui-router.rule=PathPrefix(`/apiman`, `/apimanui`)
+      - traefik.http.routers.apimanui-router.service=apimanui-service
+      - traefik.http.services.apimanui-service.loadbalancer.server.port=8080
+    restart: always
+
+  gateway:
+    image: 'apiman/vertx-gateway:latest'
+    container_name: gateway
+    environment:
+      - >-
+        APIMAN_GATEWAY_OPTS=-Dapiman.es.host=${ELASTICSEARCH_HOST}
+        -Dapiman.es.protocol=${ELASTICSEARCH_PROTOCOL}
+        -Dapiman.es.port=${ELASTICSEARCH_PORT}
+        -Dapiman.es.timeout=10000
+        -Dapiman.es.username=${ELASTICSEARCH_USERNAME}
+        -Dapiman.es.password=${ELASTICSEARCH_PASSWORD}
+        -Dapiman.truststore-password=${TRUSTSTORE_KEYSTORE_PASSWORD}
+        -DKEYCLOAK_URL=${KEYCLOAK_URL}
+        -Dkeycloak.realm=apiman
+        -DallowSelfSigned=${SELF_SIGNED}
+        -DallowAnyHost=${SELF_SIGNED}
+        -Dkeycloak.realm-public-key=${KEYCLOAK_REALM_PUBLIC_KEY}
+        -Dkeycloak.gateway.secret=${KEYCLOAK_GATEWAY_SECRET}
+        -DcachingPolicy.maxCacheSize=${MAX_CACHE_SIZE_IN_MB}
+        -Dapiman.gateway-backend-keystore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_BACKEND_KEYSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-backend-truststore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_BACKEND_TRUSTSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-client-keystore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_CLIENT_KEYSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-client-truststore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_CLIENT_TRUSTSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-es-keystore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_ES_KEYSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-es-truststore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_ES_TRUSTSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-keycloak-keystore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_KEYCLOAK_KEYSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-keycloak-truststore=${KEYSTORE_TRUSTSTORE_PATH:-}${GATEWAY_KEYCLOAK_TRUSTSTORE:-/usr/src/apiman/apiman-distro-vertx/certs/apiman.jks}
+        -Dapiman.gateway-allowedProtocols=${TLS_ALLOWED_PROTOCOLS:-}
+        -Dapiman.public-endpoint=${ENDPOINT}:80/gateway
+        -Dkeycloak.protocol=${KEYCLOAK_PROTOCOL}
+        -Dkeycloak.host=${KEYCLOAK_HOST}
+        -Dkeycloak.port=${KEYCLOAK_PORT}
+#    ports:
+#      - '${GATEWAY_PORT}:${GATEWAY_PORT}'
```

Comment:
Not sure about that line.
Why do we change the port in the container as well?
We may have to look at the conf-es.json.